release/ansible/setup.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import setuptools
+
+requirements = [
+    'kubernetes==25.3.0',
+    'oauthlib==2.1.0',
+    'requests-oauthlib==1.0.0',
+    'jmespath==0.9.0',
+]
+
+dependency_links = [
+]
+
+test_requirements = [
+    # TODO: put package test requirements here
+]
+
+setuptools.setup(
+    name='ose-ansible-operator',
+    version='4.14',
```

Comment:
Is there an anticipation that ART will be able to automatically bump this version as necessary? Not the end of the world, but it is nice that ART has some automation to handle these OCP version bumps for us instead of us doing it manually every release

Reply:
I'm honestly not too sure. There does seem to be a fair amount of automation around version bumps, but hopefully @joepvd could comment on this and let us know if there is something else I should do here. From what I can tell cachito only really needs the name and version fields to get this to pass, but that is definitely a value we need to make sure keeps in sync.